src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoSortableDateTimePattern.cs

```diff
@@ -10,19 +11,18 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoSortableDateTimePattern
     {
-        // PosTest1: Call SortableDateTimePattern getter method should return correct value for InvariantInfo
-        [Fact]
-        public void TestGetter()
-        {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo, "yyyy'-'MM'-'dd'T'HH':'mm':'ss");
-            VerificationHelper(new CultureInfo("en-us").DateTimeFormat, "yyyy'-'MM'-'dd'T'HH':'mm':'ss");
-            VerificationHelper(new CultureInfo("ja-JP").DateTimeFormat, "yyyy'-'MM'-'dd'T'HH':'mm':'ss");
+        public static IEnumerable<object[]> SortableDateTimePattern_TestData()
+        {            
```

Comment:
The latest commit brought some more extraneous trailing whitespace in.  `git diff --check` shows unstaged trailing whitespace, `git diff --check --cached` shows staged; and it looks like `git diff HEAD~2 --check` will show it for your last two commits